### ISSUE TITLE
Multiple channels with same name 

### ISSUE THREAD

**Issue by privatesam** [Cigaras/IPTV.bundle#78]
I've noticed if I have multiple channels with the exact name even if they have different URLs they appear as one entry in the playlist - I'm gonna have a look at the code and see if I can rectify.

**Comment by Cigaras** [comment]
This is Plex issue, I just tried it, plug-in adds two VideoClipOpbject objects to an ObjectContainer and passes it to Plex who for unknown reasons distinct them. This can be verified by passing container to the [len()](https://docs.python.org/3/library/functions.html#len) function ir MainMenu() or ListItems() right before returning it, like this:
```python
Log.Debug('number of objects in container: %s' % len(oc))
return oc
```

**Comment by Cigaras** [comment]
Fixed in version [2.0.0](https://github.com/Cigaras/IPTV.bundle/releases/tag/v2.0.0)
